src/modules/accounting/types.py

```diff
@@ -61,7 +63,7 @@ class AccountingProcessingState:
 
 @dataclass
 class OracleReportLimits:
-    churn_validators_per_day_limit: int
+    exited_validators_per_day_limit: int
```

Comment:
Is this change backward compatible?

Reply:
Yes, because
```
    @classmethod
    def from_response(cls, **kwargs) -> Self:
        # Compatability breaking rename
        # churn_validators_per_day_limit -> exited_validators_per_day_limit
        # Unpack structure by order
        return cls(*kwargs.values())  # pylint: disable=no-value-for-parameter
```